src/System.Private.Xml/src/System/Xml/Xsl/XsltOld/ContainerAction.cs

```diff
@@ -1019,7 +1019,6 @@ internal override void ReplaceNamespaceAlias(Compiler compiler)
             {
                 return;
             }
-            int count = this.containedActions.Count;
             for (int i = 0; i < this.containedActions.Count; i++)
```

Comment:
Isn't it better to change the `this.containedActions.Count;` in the condition with this count variable

Reply:
I can if reviewers prefer. I was aiming to simply remove dead code, using the count variable technically could change behavior (for instance if the list is being modified during enumeration).

Reply:
hm if its expected that list could be modified during loop that is strange, anyways no need to change thank you. 

XML section looks good to me